src/sentry/api/endpoints/user_authenticator_details.py

```diff
@@ -201,13 +201,13 @@ def delete(self, request: Request, user, auth_id, interface_device_id=None) -> R
                             send_email=False,
                         )
 
-            capture_security_activity(
-                account=user,
-                type="mfa-removed",
-                actor=request.user,
-                ip_address=request.META["REMOTE_ADDR"],
-                context={"authenticator": authenticator},
-                send_email=not interface.is_backup_interface,
-            )
+        capture_security_activity(
+            account=user,
+            type="mfa-removed",
+            actor=request.user,
+            ip_address=request.META["REMOTE_ADDR"],
+            context={"authenticator": authenticator},
+            send_email=not interface.is_backup_interface,
+        )
```

Comment:
Do we also need to worry about the `capture_security_activity` call at line 195? I considered this fix and was worried that the unit tests just weren't exposing it if `backup_interfaces` happens to be empty in all test cases.

Reply:
with the new options-boundary thing in place I'm going to back out the manual changes since (imo) they make the code worse